plugins/generator-1.16.5/forge-1.16.5/cmdargs/number_default.java.ftl

```diff
@@ -0,0 +1 @@
+.then(Commands.argument("${field$name}", DoubleArgumentType.doubleArg())<#if statement$args??>${statement$args}</#if>)
```

Comment:
Why do these arguments have

`<#if statement$args??>`

while other arguments don't? Is this only needed here for some reason, or is it needed in all cases, or not needed at all?

Reply:
Why was this in the first place? I wanted to know and you just closed this issue.

Reply:
I don't remember why I added it (but it's not needed). I guess I first added them because I based all my work on the AI tasks editor, but then, I didn't add them to other blocks (Number was the first block I made/tested).